lib/CleantalkSP/SpbctWP/Scanner/ScannerQueue.php

```diff
@@ -999,6 +1000,10 @@ public function signature_analysis($status = 'UNKNOWN,MODIFIED,OK,INFECTED', $of
                     . ' WHERE fast_hash = \'' . $file['fast_hash'] . '\';'
                 );
 
+                // Added scan result to table with log
```

Comment:
Please, add the todo to make one bulk SQL-request instead of  tiny multiple requests.

Reply:
OK